components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -0,0 +1,692 @@
+---
```

Comment:
please change double quotes to single quotes everywhere (as long as not inside a string)